docs/application/dotnet/guides/applications/component-based-app.md

```diff
@@ -0,0 +1,607 @@
+# Component Based Applications
```

Comment:
We follow sentence style for capitalization for new guides, hence request to change all headings to:
Component based applications 

Grammatically it should be "Component-based applications" however since it is API name I am not recommending a hypen.